src/main/frontend/client/components/BerriesIndex.js

```diff
@@ -29,12 +29,7 @@ const BerriesIndex = props => {
     return <BerriesTile key={berries.id} berries={berries} />
```

Comment:
We will then need to update this to `BerryTile` as well once the import has changed.

Also just noticed that within `berries.map` we are calling each element the plural `berries`.  We can instead change that to the singular `berry` to be more descriptive.